localnet/single/template/setup.sh

```diff
@@ -12,6 +12,7 @@ CHAIN_ID="simd-testing"
 MONIKER="simd-testing"
 CONFIG_DIR=${SCRIPT_DIR}/.finschia
 CHAIN_DIR=${CONFIG_DIR}
+DEMON="cony"
```

Comment:
Oh It'is typo, isn't it? 
Did you intend to type DENOM?

Reply:
Oh, that's my typo mistake, I'll fix it. Thank you very much.